runtime/README.md

```diff
@@ -692,107 +708,504 @@ Command Code: `0x434B_4558` ("CKEX")
+| input size | u32      | size of input   |
+| input      | u8[...]  |                 |
+
+*Table: `CM_RANDOM_STIR` output arguments*
+| **Name**    | **Type** | **Description**              |
+| ----------- | -------- | ---------------------------- |
+| chksum      | u32      |                              |
+| fips_status | u32      | FIPS approved or an error    |
+
+### CM_RANDOM_GENERATE
+
+This generates random bytes that are returned from the internal RNG.
+
+Command Code: `0x434D_5247` ("CMRG")
+
+*Table: `CM_RANDOM_GENERATE` input arguments*
+
+| **Name**            | **Type** | **Description** |
+| ------------------- | -------- | --------------- |
+| chksum              | u32      |                 |
+| data size to return | u32      |                 |
+
+
+*Table: `CM_RANDOM_GENERATE` output arguments*
+| **Name**    | **Type** | **Description**           |
+| ----------- | -------- | ------------------------- |
+| chksum      | u32      |                           |
+| fips_status | u32      | FIPS approved or an error |
+| output size | u32      | size of output            |
+| output      | u8[...]  |                           |
+
+### CM_IMPORT
+
+Imports the specified data and returns a CMID for it.
+
+Usage information is required so that the data can be verified and used appropriately.
+
+Command Code: `0x434D_494D` ("CMIM")
+
+*Table: `CM_IMPORT` input arguments*
+
+| **Name**   | **Type** | **Description**                            |
+| ---------- | -------- | ------------------------------------------ |
+| chksum     | u32      |                                            |
+| usage      | u32      | Tag to specify how the data can be used    |
+|            |          | `CMGE` - AES-256-GCM key, must be 32 bytes |
+|            |          | `HMAC` - Can be used for HMAC, HKDF        |
+| input size | u32      |                                            |
+| input      | u8[...]  |                                            |
+
+*Table: `CM_IMPORT` output arguments*
+| **Name**    | **Type** | **Description**              |
+| ----------- | -------- | ---------------------------- |
+| chksum      | u32      |                              |
+| fips_status | u32      | FIPS approved or an error    |
+| CMID        | u8[32]   | CMID containing imported key |
+
+### CM_DELETE
+
+Deletes the object stored with the given mailbox ID.
+
+Command Code: `0x434D_444C` ("CMDL")
+
+*Table: `CM_DELETE` input arguments*
+
+| **Name** | **Type** | **Description**            |
+| -------- | -------- | -------------------------- |
+| chksum   | u32      |                            |
+| CMID     | u8[32]   | CMID of the data to delete |
+
+*Table: `CM_DELETE` output arguments*
+| **Name**    | **Type** | **Description**              |
+| ----------- | -------- | ---------------------------- |
+| chksum      | u32      |                              |
+| fips_status | u32      | FIPS approved or an error    |
+
+### CM_CLEAR
```

Comment:
I wonder if it would be helpful to report how much CM storage is in use in some status command.

Reply:
That's a good idea. I'll add that.